docs/reference/presigned-url.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: default
+title: Presigned URL
+description: Configuring lakeFS to use presigned URLs.
+parent: Reference
+nav_order: 100
+has_children: false
+---
+
+# Presigned URL
+{: .no_toc }
+
+{% include toc.html %}
+
+By default, lakeFS will get and set objects using its server. lakeFS also support accessing objects using a presinged URL which allows direct access to the object.
+lakeFS assume by default your object storage support presinged URLs, if not it should be disabled in the configuration.
```

Comment:
Do you mean the UI?

Reply:
No, if the object storage doesn't support presigned URLs in general, like MinIO